src/jit/jitee.h

```diff
@@ -0,0 +1,259 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+// This class wraps the CORJIT_FLAGS type in the JIT-EE interface (in corjit.h) such that the JIT can
+// build with either the old flags (COR_JIT_EE_VERSION <= 460) or the new flags (COR_JIT_EE_VERSION > 460).
+// It actually is exactly the same as the new definition, and must be kept up-to-date with the new definition.
```

Comment:
The issue is we must be able to build with either COR_JIT_EE_VERSION <= 460, and the old CORJIT_FLAGS type definition (a struct of 2 dwords and all the old CorJitFlag enum declarations) -- this is the CTP JIT -- or with COR_JIT_EE_VERSION > 460 and the new CORJIT_FLAGS type definition (my new 'class') and new enum values. Probably, after we ship the next desktop update, we can assume that any future CTP desktop JIT would run on top of that, and we can get rid of all COR_JIT_EE_VERSION `#ifdefs` and subsequent JIT-EE interface changes would be made as `#if COR_JIT_EE_VERSION > 463` (for example). That would allow us to eliminate this adapter class if desired.